packages/block-library/src/verse/editor.scss

```diff
@@ -1,3 +0,0 @@
-pre.wp-block-verse {
-	color: $gray-900;
```

Comment:
This is probably okay to remove. It didn't look like we were applying this to the published view. Not sure why we needed this one.